changelog/29138.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:improvement
```

Comment:
Since the change is specific to docs update only, changelog entry is not required.